src/lib/containers/QueryWrapper.tsx

```diff
@@ -134,6 +135,7 @@ export default class QueryWrapper extends React.Component<
     this.getNextPageOfData = this.getNextPageOfData.bind(this)
     this.updateParentState = this.updateParentState.bind(this)
     this.getInitQueryRequest = this.getInitQueryRequest.bind(this)
+    const showFacetVisualization = props.defaultShowFacetVisualization ?? true
```

Comment:
Just curious, where do you use `showFacetVisualization` in the code?

Reply:
Down on line 156 in this file.  It used to always set to true, but will now use the input param (if set)